cyberdrop_dl/managers/args_manager.py

```diff
@@ -104,7 +104,7 @@ def startup(self) -> None:
             self.log_dir = Path(self.parsed_args['log_folder'])
         if self.parsed_args['sort_downloads']:
             self.sort_downloads = True
-        if not self.parsed_args['sort_cdl_only']:
+        if self.parsed_args['sort_all_downloads']:
```

Comment:
this broke my app, seems like my settings created only `sort_cdl_only` and not `sort_all_downloads`